src/mower_logic/src/mower_logic/behaviors/UndockingBehavior.cpp

```diff
@@ -55,37 +55,45 @@ Behavior *UndockingBehavior::execute() {
   docking_pose_stamped_front.pose = pose.pose.pose;
   docking_pose_stamped_front.header = pose.header;
 
-  int undock_point_count = config.undock_distance * 10.0;
+  int undock_point_count = 10;
```

Comment:
Giving the `10` a name for clarity is fine for me, but it should be `const` then.

Reply:
Done.